### ISSUE TITLE
[New Tip] Design tip no2 - Example

### ISSUE THREAD

**Issue by dboudreau** [w3c/wai-website#743]
**Reference:** http://w3c.github.io/wai-quick-start/designing.html#dont-use-color-alone-to-convey-information

I suggest we add additional information for screen reader users to understand the examples better. Currently, the `<figcaption>` text for the failed examples read:

```
<figcaption>
<svg class="cross" aria-hidden="true">
<use xlink:href="images/icons.svg#cross">
</svg>
Color only
</figcaption>
```

I suggest we provide additional context:

```
<figcaption>
<svg class="cross" aria-hidden="true">
<use xlink:href="images/icons.svg#cross">
</svg>
<div class="hidden-accessible">Bad example using</div>
Color only
</figcaption>
```

Likewise, the examples for valid implementation read:

```
<figcaption>
<svg class="tick" aria-hidden="true">
<use xlink:href="images/icons.svg#tick">
</svg>
Color and symbol
</figcaption>
```

I suggest we provide additional context:

```
<figcaption>
<svg class="tick" aria-hidden="true">
<use xlink:href="images/icons.svg#tick">
</svg>
<div class="hidden-accessible">Good example using</div>
Color and symbol
</figcaption>
```

Where "hidden-accessible" would refer to the following CSS rules:

```
.hidden-accessible {
     border: 0; 
     clip: rect(0 0 0 0); 
     height: 1px; 
     width: 1px; 
     margin: -1px; 
     padding: 0; 
     overflow: hidden; 
     position: absolute;
}
```
